epoch/api/channels_api_usage.md

```diff
@@ -451,9 +459,18 @@ it and then post it back via a WebSocket message:
 }
 ```
 
+#### Signed channel_close_mutual_tx
+Both participans receive the co-signed `channel_close_mutual_tx`:
+```
+{'action': 'on_chain_tx',
+ 'payload': {'tx': 'tx$T7f9gwgv5SsoFi3uoKcjZcfG9wXqS52nRHiCZccmc2pYAxc17sfDDNu6EAVgxQQxD2pvbQGBBKpatdkczSQWPxqMdXpgPgKEqEGxzmqJmeNMFR81KotMjPAiJciuinTQ1Gr2jLnfck3mGhnpXT9DrniEYdqnd53nP2XGMPsodRC8dUUMFzANh3YzBn3xsrNA1eHVBu4SZVMGhUHR3baVGKNpdHhsz8RiYTc6bjHBDhRiRPtgfvF4swJGDUsgWpAciLnssdq8AZ9cVy5JyReg3mf8UJaSLfvVUATyN9LN3Jf3pW'
+            }
+}
+```
+Using its hash participants can track its progress on the chain - entering the mempool, block inclusion and a number of confirmations.
```

Comment:
"Using its hash, participants can track its progress on the chain: entering the mempool, block inclusion ..."